.github/workflows/docs.yml

```diff
@@ -3,7 +3,9 @@ name: docs
 on:
   push:
     branches: [ master ]
-  pull_request:
+    paths:
```

Comment:
Leave the `pull_request` as it is because the readthedocs project is not set up for build & preview. Try to finish one thing for one PR.